### PR TITLE
Par-Test allows redefinition of srcDir by Ant

### PR DESCRIPTION
**Pull request by som-snytt**
Since srcDir is no longer used to extract the test kind from any
given test path, it is especially benign to rely on the defness
of srcDir in order to allow redefinition during an Ant run (as
required by test.scaladoc).

The parameter is configured via the partest.srcdir property.

Maybe the keyword should be def for definite values and
red for redefinable values and ind for indefinite values.
And ded for dedefinable values optionally reset to None.

Review by @hubertp who spotted it

### CODE REVIEW

File: src/partest/scala/tools/partest/nest/PathSettings.scala [hunk 1]
```diff
@@ -82,7 +82,7 @@ object PathSettings {
    *  if one of those environment variables is set, then the lib directory under java.home,
    *  and finally the lib directory under the parent of java.home. Or, as a last resort,
    *  search deeply under those locations (except for the parent of java.home, on the notion
-   *  that if this is not a canonical installation, then that search would have litte
+   *  that if this is not a canonical installation, then that search would have little
```
Comment (paulp) [inline src/partest/scala/tools/partest/nest/PathSettings.scala:85]: Spelling corrections should be in a separate pull request.

Ha ha just being hilarious.

**Comment by paulp** [comment]
LGTM

File: src/partest/scala/tools/partest/nest/PathSettings.scala (hunk 1 above)
Comment (som-snytt) [inline src/partest/scala/tools/partest/nest/PathSettings.scala:85]: You got me.
